integration/cluster_test.go

```diff
@@ -234,19 +244,29 @@ func NewCluster(t *testing.T, size int) *cluster {
 	return c
 }
 
+// NewCluster returns an unlaunched cluster of the given size which has been
+// set to use static bootstrap.
+func NewCluster(t *testing.T, size int) *cluster {
+	return newCluster(t, size, false)
+}
+
 // NewClusterUsingDiscovery returns an unlaunched cluster of the given size
 // which has been set to use the given url as discovery service to bootstrap.
 func NewClusterByDiscovery(t *testing.T, size int, url string) *cluster {
 	c := &cluster{}
 	ms := make([]*member, size)
 	for i := 0; i < size; i++ {
-		ms[i] = mustNewMember(t, c.name(i))
+		ms[i] = mustNewMember(t, c.name(i), false)
 		ms[i].DiscoveryURL = url
 	}
 	c.Members = ms
 	return c
 }
 
+func NewTLSCluster(t *testing.T, size int) *cluster {
```

Comment:
NewTLSClusterByDisocvery ensures the member publish the correct https scheme to discovery service and discovery service also returns back the correct https scheme.